README.md

```diff
@@ -2,3 +2,33 @@
 
 This repository contains the scripts and other materials needed to execute the
 workshop for SIG Automation at the 2022 ONE Summit in Seattle.
+
+## Setup Details
+
+### Virtual Machine Details
+
+| <!-- -->         | <!-- -->     |
+|------------------|--------------|
+| Operating System | Ubuntu 22.04 |
+| Architecture     | x86_64       |
+| Cores            | 8            |
+| RAM              | 32 GB        |
+| Disk             | TBD          |
+
+
+
+### Pre-Installed Software
+- docker
+- git
+-
+
+### KIND Clusters
+KIND Version -> TBD
+
+| Name     | Description | Control Nodes | Worker Nodes | CNIs | Operators |
```

Comment:
In case that we decided to use different k8s versions per cluster.
```suggestion
| Name     | Description | K8s Version |Control Nodes | Worker Nodes | CNIs | Operators |
```

Reply:
oh I don't think we should do that in this timeframe.